### PR TITLE
Add a very basic PSF announcement post

### PR DESCRIPTION
**Pull request by ericholscher**
I didn't quite know what to write about it,
since it feels very salesy.
It feels like we almost want a case study or something,
but I don't think we're quite ready for that.

Would love some more suggestions of what would be good content/context to add.

### CODE REVIEW

File: content/posts/psf-ethicalads-partnership.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+Title: Python Software Foundation & EthicalAds partnership announcement
+Date: May 2, 2022
+description: We're excited to announce our partnership to help the PSF promote their sponsors across their ecosystem of web properties.
+tags: publishers, sponsorship, partnership, announcement
+authors: Eric Holscher
+image: /theme/img/python-code.jpg
+
+We're excited to announce a partnership with the [Python Software Foundation](https://www.pythoneorg/psf/) (PSF).
```
Comment (davidfischer) [inline content/posts/psf-ethicalads-partnership.md:8]:
```suggestion
We're excited to announce a partnership with the [Python Software Foundation](https://www.python.org/psf/) (PSF).
```

File: content/posts/psf-ethicalads-partnership.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+They started with the sites that are already included in their sponsorship prospectus.
+The properties we are currently implementing sponsorship on are:
+
+* The [Python Package Index](https://pypi.org/project/urllib3/1.26.7/#changes)
+* The [PSF sponsor listing page](https://www.python.org/psf/sponsors/)
+
+You can see the PyPI integration looks similar to our normal ad placements,
+whereas the PSF sponsors page has a custom look and feel.
+We are able to customize each of the placements used in our sponsorship platform,
+so they can look exactly what you need.
```
Comment (davidfischer) [inline content/posts/psf-ethicalads-partnership.md:34]: "you" seems awkward in this sentence.

**Review by davidfischer — COMMENTED** [review]
> it feels very salesy.

To some degree, that means you're doing it right!

File: content/posts/psf-ethicalads-partnership.md (hunk 1 above)
Comment (ericholscher) [reply content/posts/psf-ethicalads-partnership.md:8]: Weird, not sure how that happened :(

**Comment by ericholscher** [comment]
@davidfischer This has been updated with an approved quote. It's not super compelling, but at least seems worth shipping to have a note of it.

File: content/posts/psf-ethicalads-partnership.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+Title: Python Software Foundation & EthicalAds working to promote open source sponsorship
+Date: May 2, 2022
```
Comment (davidfischer) [inline content/posts/psf-ethicalads-partnership.md:2]:
```suggestion
Date: November 1, 2022
```

Ship it on Tuesday?

**Review by davidfischer — APPROVED** [review]
Looks good. The sponsorship platform isn't just something people can "sign up" for but I guess we can pitch it like that and handle responses.

**Pull request merged**